### PR TITLE
Create JIRA issues from unmatched GitHub issues

### PR DESCRIPTION
**Pull request by LyonesGamer**
This PR builds on #1, and is thus blocked on it.

This PR adds the createIssue() function, which creates a new JIRA issue using a given GitHub issue's title and description and the project passed in the config. It also sets the custom fields for the GitHub issue ID, the issue Number, the GitHub labels, and the time the tool ran.

The function does not set the Reporter, Assignees, Issue Type, or Priority. These may be set in a future change, but for now, they default to the user under which the tool runs, nobody, Task, and Medium, respectively.

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -1,61 +1,420 @@
...
+
+	return nil
+}
+
+func updateIssue(ghIssue github.Issue, jIssue jira.Issue, client jira.Client) error {
+	log.Debugf("Updating JIRA %s with GitHub #%d", jIssue.Key, *ghIssue.Number)
+	return nil
+}
+
+func createIssue(issue github.Issue, client jira.Client) error {
```
Comment (squat) [inline cmd/root.go]: This is the only change here that's different from the last PR, right?

Comment (LyonesGamer) [inline cmd/root.go]: Correct.

**Comment by LyonesGamer** [comment]
@squat I understand what you mean about moving it all into a separate file. Now that you point it out, I agree; this is doing far too much in one file. That may need to wait for a later PR, though, since all of the PRs work on these functions in this file.

**Comment by squat** [comment]
It's not just too much for one file but simply that this code does not belong in this package at all.

**Comment by LyonesGamer** [comment]
I completely agree.

**Pull request merged**